scripts/startLocalCluster.sh

```diff
@@ -87,15 +87,15 @@ SFRE=1.0
 COMP=500
 KEEP=2000
 if [ -z "$ONGOING_PORTS" ] ; then
-  CO_BASE=$(( $PORT_OFFSET + 8530 ))
+  CO_BASE=$(( $PORT_OFFSET + 8529 ))
```

Comment:
ok, I know why you committed this, but can we not commit it?
This will now start Arango for everyone on other ports.

Reply:
Knew that this will be found :) 
Will revert that change (but also e.g. arangodb starter is using 8529 instead of 8530 etc.)